platform/security/rest/security-rest-cxfwrapper/src/main/java/org/codice/ddf/cxf/oauth/OAuthSecurityImpl.java

```diff
@@ -324,6 +326,7 @@ private String getNewAccessToken(
 
     Form formParam = new Form(GRANT_TYPE, grantType);
     queryParameters.forEach(formParam::param);
+    formParam.param(SCOPE, OPENID_SCOPE);
```

Comment:
I already approved, but we have a configuration for OAuth handler which determines the scopes that will be requested from the auth server. Wondering if we should include the full set from that config here as well

Reply:
The problem is those that configuration is separate. This code is for federating and that is for login so they could be connected to two different providers.

Reply:
Do we see a use case where these federation scopes would need to be configurable? If so, I would imagine that would live in the source configuration somewhere, but 🤷‍♂

Reply:
Actually, it probably is fine as-is. It's likely we'll hit a situation where additional scopes are needed, but auth servers should be configurable to provide scopes as needed to specific clients without the client requesting them.

Reply:
If that is the case we will need an additional configuration in the source configurations to take care of that.

Reply:
I was trying to say we shouldn't need to do that 😄 It makes sense to request the "openid" scope since that's required for OIDC, but for application-specific scopes, it should be possible to configure everything in the auth server rather than DDF. That's at least the case for Keycloak. You can provide a list of default scopes to issue on a per-client basis.